CHANGELOG.md

```diff
@@ -68,6 +68,9 @@ Thanks to [#5342](https://github.com/decidim/decidim/pull/5342), Decidim now sup
 
 **Fixed**:
 
+- **decidim-admin**: Avoid user_manager permissions to shadow space admin permissions. [\#5698](https://github.com/decidim/decidim/pull/5698)
+- **decidim-core**: Fix: display the correct google brand log in omniauth login view. [\#5685](https://github.com/decidim/decidim/pull/5685)
```

Comment:
Are these changes intended? Maybe they come from older code?

Reply:
You are right, there's a duplicated line that appeared when resolving the typo in master's changelog.

Reply:
This line is still duplicated! 😄